src/functions/phantom.js

```diff
@@ -0,0 +1,36 @@
+import defineFunction, {ordargument} from "../defineFunction";
+import buildCommon from "../buildCommon";
+import mathMLTree from "../mathMLTree";
+
+import * as html from "../buildHTML";
+import * as mml from "../buildMathML";
+
+defineFunction(
+    "\\phantom",
+    {
+        numArgs: 1,
+    },
+    (context, args) => {
+        const body = args[0];
+        return {
+            type: "phantom",
+            value: ordargument(body),
+        };
+    },
+    "phantom",
```

Comment:
It's coincidental that the name of the function and the type matches, most of the time this won't be the case.